Automattic/wp-calypso
# Colour Scheming: Update Checklist Colours

#### Changes proposed in this Pull Request

* Switches some HEX codes to the HEX for the success colour: `#008a00`

#### Testing instructions

Prompt the checklist and check the gauge (I changed the colours based on searching the HEX, and to be honest, I didn't even see that SVG in the original image when I checked the checklist - I think it depends on the step).

Question: should green in the SVGs use `#008a00`? It'd probably be worth checking the other SVGs the checklist uses if so

Fixes #31649

cc @flootr, @blowery, @drw158, @sixhours
